dockerfiles/services/request-replayer/src/index.php

```diff
@@ -59,13 +59,14 @@ function logRequest($message, $data = '')
         break;
     default:
         $headers = getallheaders();
-        if (isset($headers['X-Datadog-Diagnostic-Check'])) {
+        if (isset($headers['X-Datadog-Diagnostic-Check']) || isset($headers['x-datadog-diagnostic-check'])) {
```

Comment:
Just noting libdatadog uses the fact these are case insensitive, but the request replayer isn't currently normalizing them, so it was quick and easy to just check the two variants we get.